src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_TYPES = 5;
+    public static final int MAX_SIDE_LEN = 100;
+    public static final int RADIUS = 10;
+    public static final Color DEFAULT_COL = Color.WHITE;
```

Comment:
1. Do not shorten words
2. **RADIUS** is not an informative name. Change to **DEFAULT_RADIUS**